docs/restic_troubleshooting.md

```diff
@@ -0,0 +1,365 @@
+apiVersion: v1
+data:
+  AWS_ACCESS_KEY_ID: QUtJQVZCUsnip
+  AWS_DEFAULT_REGION: dXMtdsnip
+  AWS_SECRET_ACCESS_KEY: ZGZQsnip
+  RESTIC_PASSWORD: cmVzdGljcGFzc3dvcmQ=
+  RESTIC_REPOSITORY: czM6czMuYW1hem9uYXdzLmNvbS9jdnBidWNrZXR1c3dlc3Qy
+  restic-prune-interval: MQ==
+kind: Secret
+metadata:
+  creationTimestamp: "2023-06-14T17:53:41Z"
+  labels:
+    openshift.io/oadp: "True"
+    openshift.io/oadp-bsl-name: dpa-sample-1
+    openshift.io/oadp-bsl-provider: aws
+  name: dpa-sample-1-volsync-restic
+  namespace: openshift-adp
+  ownerReferences:
+  - apiVersion: oadp.openshift.io/v1alpha1
+    blockOwnerDeletion: true
+    controller: true
+    kind: DataProtectionApplication
+    name: dpa-sample
+    uid: 66568a80-778a-4478-bca1-d8ff7720b129
+  resourceVersion: "28139203"
+  uid: 192bc903-e754-4cd3-9173-2af805c2b0d0
+type: Opaque
+```
+
+#### decode the restic passwd 
+```
+cho "cmVzdGljcGFzc3dvcmQ=" | base64 -d
+resticpassword
+```
+
+#### datamover restic path 
+
+The path in 1.2.0 is
+`$bucket/openshift-adp/$snapcontent_name`
+
+The snapcontent_name = sourcePVC
+
+```
+spec:
+  restic:
+    accessModes:
+    - ReadWriteOnce
+    copyMethod: Direct
+    customCA: {}
+    moverServiceAccount: velero
+    pruneIntervalDays: 1
+    repository: vsb-zg6gg-secret
+    retain: {}
+    storageClassName: gp2-csi
+    volumeSnapshotClassName: csi-aws-vsc-test
+  sourcePVC: snapcontent-2044fb64-253d-461b-93f3-1ce8d6b67ebe-pvc
+```
+
+#### list snapshots for DataMover restic snapshot
+
+```
+restic  --cache-dir /tmp/.cache -r s3:s3-us-west-2.amazonaws.com/cvpbucketuswest2/openshift-adp/snapcontent-993aabe2-8170-4661-984e-00a560f486cd-pvc snapshots
+enter password for repository: 
+repository 85c55159 opened (version 2, compression level auto)
+created new cache in /tmp/.cache
+ID        Time                 Host        Tags        Paths
+------------------------------------------------------------
+dcec01b1  2023-06-20 20:16:46  volsync                 /data
+------------------------------------------------------------
+1 snapshots
+```
+
+##  Update DPA for retain policy - restic forget
+```
+  features:
+    dataMover:
+      credentialName: restic-secret
+      enable: true
+      pruneInterval: "1"
+      snapshotRetainPolicy:
+        hourly: "1"
+```
+
+## Run a new backup and check replicationsource
+
+```
+oc get replicationsource vsb-zg6gg-rep-src -n openshift-adp -o yaml
+apiVersion: volsync.backube/v1alpha1
+kind: ReplicationSource
+metadata:
+  creationTimestamp: "2023-06-20T21:04:28Z"
+  generation: 1
+  labels:
+    datamover.oadp.openshift.io/vsb: vsb-zg6gg
+  name: vsb-zg6gg-rep-src
+  namespace: openshift-adp
+  resourceVersion: "28168858"
+  uid: 53dc160a-d0c1-416a-95fb-77f316e8e0c1
+spec:
+  restic:
+    accessModes:
+    - ReadWriteOnce
+    copyMethod: Direct
+    customCA: {}
+    moverServiceAccount: velero
+    pruneIntervalDays: 1
+    repository: vsb-zg6gg-secret
+```
+
+#### get snapshots
+```
```

Comment:
I would separate commands and outputs for each part

For example: here something like

For a repository located at `<REPOSITORY-URL>`, run
```
restic --cache-dir /tmp/.cache -r <REPOSITORY-URL> snapshots
```
After typing your repository password, you should get something like
```
repository 83b7f53a opened (version 2, compression level auto)
ID        Time                 Host        Tags        Paths
------------------------------------------------------------
ab60e48b  2023-06-20 21:04:41  volsync                 /data
------------------------------------------------------------
1 snapshots
```

Reply:
++ true, that was lazy of me.